### DISCUSSION TITLE
Calling a exported function of a component CSF3

### DISCUSSION THREAD

**Discussion by cbishop-fv** [storybookjs/storybook#26914] [Help]
### Summary

I'm trying to test a svelte component using CSF3. The component is a snack bar that is not shown until the exported open function is called. How, in CSF3, can I call an exported method?

```
<script lang="ts">
    export let text= '';

    export const open = () => {
        ...
    };
</script>

<Modal>
    ...
```

calling open in this example

### Additional information

Storybook 7.6.17  
Svelte 4

**Comment by SaschaLucius** [comment]
@cbishop-fv have you figured it out?
